src/DynamoCore/Graph/Nodes/CodeBlockNode.cs

```diff
@@ -377,18 +377,24 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
             shouldFocus = helper.ReadBoolean("ShouldFocus");
             code = helper.ReadString("CodeText");
 
+            var inportConnections = new OrderedDictionary();
+            var outportConnections = new OrderedDictionary();
+            SaveAndDeleteConnectors(inportConnections, outportConnections);
+
```

Comment:
comments please on why this was left out in previous PR or why this is required here.

Reply:
yep, will add a comment in code - but previous to Ian's refactor we didn't actually make new portModels we just updated them, but he removed the data property of ports, so now new models are created instead, so we have to delete and create new connectors to go along with those ports.